frontend/src/pages/DashboardPage/Dashboard/Dashboard.css

```diff
@@ -187,8 +158,9 @@
 
 /* Highlighted button */
 .active-button {
```

Comment:
What is the active-button for?

Reply:
Its CSS for the time button user is currently at (active)